packages/normalizr/src/denormalize.ts

```diff
@@ -37,6 +37,16 @@ const unvisitEntity = (
   if (entity === DELETED) {
     return [undefined, true, true];
   }
+  if (
+    process.env.NODE_ENV !== 'production' &&
+    typeof entity === 'symbol' &&
+    (entity as symbol).toString().includes('DELETED')
+  ) {
+    throw new Error(
+      `Unrecognized symbol detected.
+Make sure you do not have multiple versions of @rest-hooks/normalizr installed.`,
```

Comment:
the solution here is that normalizr should be a peer dep everywhere it's used - this kind of check is the same kind of runtime protection that React does (because React should also be a peer dep).

Another option is that you could use `Symbol.for`, and then there's no state that needs to be a singleton.

Reply:
Symbol.for seems a bit scary without changing the naming. That would be a breaking change so maybe something to do in the future. Honestly I'm not sure why people get multiple normalizrs, but multiple reacts come easily when using npm since it has a tendency to just keep spawning new installs even when ranges match existing. (even for peerdeps)

Reply:
you'd definitely have to change the naming for that to work.

i've never seen any of those kinds of issues when it's consistently a peer dep, and npm 7 will fail the entire install if there's conflicting reacts ¯\\\_(ツ)\_/¯